vendor/github.com/Azure/azure-docker-extension/pkg/vmextension/handlersettings.go

```diff
@@ -9,7 +9,7 @@ import (
 	"io/ioutil"
 	"path/filepath"
 
-	"github.com/Azure/azure-docker-extension/pkg/executil"
+	"os/exec"
```

Comment:
ah fair vscode keeps failing me on imports, but I'll keep it the same as it is harmless for now.